index.html

```diff
@@ -41,15 +42,12 @@
     <body></body>
     <script>
 
-const KEY_CODE_SPACE = 32;
 
 // this function will eventually run the jsPsych timeline
-function kickOffExperiment(stimuli) {
-    const G_QUESTION_CHOICES = [FALSE_BUTTON_TEXT, TRUE_BUTTON_TEXT];
```

Comment:
Perhaps move the kickOffExperiment() to the main.js as well. Then the index.html's only job is to start the main function on the "load" event. And keep all logic related to the experiment in main.js.